src/LinuxTracing/UprobesUnwindingVisitorDwarfUnwindingTest.cpp

```diff
@@ -0,0 +1,1814 @@
+// Copyright (c) 2022 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <gmock/gmock.h>
+#include <gtest/gtest.h>
+#include <sys/mman.h>
+#include <unwindstack/Error.h>
+#include <unwindstack/MapInfo.h>
+#include <unwindstack/Unwinder.h>
+
+#include <atomic>
+#include <memory>
+#include <string>
+#include <utility>
+#include <vector>
+
+#include "GrpcProtos/capture.pb.h"
+#include "LibunwindstackUnwinder.h"
+#include "MockTracerListener.h"
+#include "PerfEvent.h"
+#include "PerfEventRecords.h"
+#include "UprobesFunctionCallManager.h"
+#include "UprobesReturnAddressManager.h"
+#include "UprobesUnwindingVisitor.h"
+#include "UprobesUnwindingVisitorTestCommon.h"
+
+namespace orbit_linux_tracing {
+
+template <typename PerfEventAndFullCallstackProtoTypePair>
+class UprobesUnwindingVisitorDwarfUnwindingTest : public ::testing::Test {
```

Comment:
May I suggest to do this the following way?

```cpp
class UprobesUnwindingVisitorDwarfUnwindingTestBase : public ::testing::Test {
// ...
};

template<typename>
class UprobesUnwindingVisitorDwarfUnwindingTest : public UprobesUnwindingVisitorDwarfUnwindingTest {};
```

This makes it a bit clearer that the test fixture does not depend on the type and it avoid code generation bloat (since the compiler doesn't need to generate the class's methods for each template type.)

Reply:
Done.